tests/integration.spec.ts

```diff
@@ -108,12 +107,23 @@ for (const packageManager of ['npm', 'pnpm', 'yarn'] as PackageManager[]) {
       expect(fs.existsSync(path.join(dir, 'playwright.config.js'))).toBeTruthy();
 
       {
-        const { code } = await exec(packageManager === 'npm' ? 'npx' : packageManager === 'pnpm' ? 'pnpm dlx' : 'yarn', ['playwright', 'install-deps']);
+        const { code } = await exec(packageManagerToNpxCommand(packageManager), ['playwright', 'install-deps']);
         expect(code).toBe(0);
       }
 
-      const { code } = await exec(packageManager === 'npm' ? 'npx' : packageManager === 'pnpm' ? 'pnpm dlx' : 'yarn', ['playwright', 'test']);
+      const { code } = await exec(packageManagerToNpxCommand(packageManager), ['playwright', 'test']);
       expect(code).toBe(0);
     });
   });
 }
+
+function packageManagerToNpxCommand(packageManager: PackageManager): string {
```

Comment:
packageManager.command would look cleaner

Reply:
this is inside the tests, would require too much wiring for adding package manager by name etc.